README.md

```diff
@@ -89,12 +89,12 @@ See [variables.tf] and [examples/] for details and use-cases.
 
 ##### VPC
 
-- **`vpc_name`**: *(Required `string`)*
+- **`vpc_name`**: *(Optional `string`)*
 
   The Name of the VPC. This will be used to tag resources with Names by default.
```

Comment:
```suggestion
  The name of the VPC. This will be used to tag resources with names by default.
```

Reply:
please do not add blocking reviews for changed out of scope.. Just create a PR.

Reply:
fixed

Reply:
got it